NodeTypes/Content/Registration/Registration.fusion

```diff
@@ -6,7 +6,7 @@ prototype(Neos.Demo:Content.Registration) < prototype(Neos.Neos:ContentComponent
     renderer = Neos.Fusion.Form:Runtime.RuntimeForm {
         namespace = "neos-demo-registration"
         attributes {
-            id = ${'form-' + node.identifier}
+            id = ${'form-' + node.property('_identifier')}
```

Comment:
as far as i know this behaviour is deprecated and `node.nodeAggregateId.value` should be used? Or we could have another flowquery operation for this? `q(node).identifier()`

Reply:
we decided to go for `q(node).id()` IIRC